docs/src/components/_overview.md

```diff
@@ -1,14 +1,16 @@
 # Valence Programs
 
-Valence Programs can be executed in two distinct ways.
+There are two ways to execute Valence Programs.
+
 1. **On-chain Execution**:
-Extensive support exists for CosmWasm and some for EVM. The rest of this section provides a high-level breakdown of the components that compose a Valence cross-chain program using on-chain coprocessors.
+Valence currently supports CosmWasm and EVM. SVM support coming soon. The rest of this section provides a high-level breakdown of the components that comprise a Valence Program using on-chain coprocessors.
     - [Domains](./domains.md)
     - [Accounts](./accounts.md)
     - [Libraries and Functions](./libraries_and_functions.md)
     - [Programs and Authorizations](./programs_and_authorizations.md)
     - [Middleware](./middleware.md)
-2. **zk-Coprocessor**:
-Early specifications exist for the [Valence zk-Coprocessor](../zk-coprocessor/_overview.md). Our current focus is to move computation off-chain as this is a more scalable approach to building a cross-chain execution environment.
 
-Unless explicitly mentioned, you may assume that documentation and examples in the remaining sections are written with on-chain execution in mind.
\ No newline at end of file
+2. **Off-chain Execution via ZK Coprocessor**:
+Early specifications exist for the [Valence ZK coprocessor] (/zk-coprocessor/overview.md). We aim to move as much computation off-chain as possible since off-chain computation is a more scalable approach to building a cross-chain execution environment.
```

Comment:
```suggestion
Early specifications exist for the [Valence ZK coprocessor] (/zk-coprocessor/_overview.md). We aim to move as much computation off-chain as possible since off-chain computation is a more scalable approach to building a cross-chain execution environment.
```